lighthouse-core/report/html/report-styles.css

```diff
@@ -993,8 +993,9 @@
 .lh-warnings--toplevel {
   --item-margin: calc(var(--header-line-height) / 4);
   color: var(--report-text-color-secondary);
-  margin: var(--category-padding);
-  padding: var(--category-padding);
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
```

Comment:
is this really what we want? feels like we should be having some sort of viewport width breakpoints to make it full size at smaller screen sizes, etc

Reply:
done - i think?

Reply:
wfm!